README.md

```diff
@@ -1,191 +1,120 @@
+
+![cncf_logo](imgs/cncf-logo.png)
 
-**Heterogeneous AI Computing Virtualization Middleware (HAMi), formerly known as k8s-vGPU-scheduler, is an "all-in-one" chart designed to manage Heterogeneous AI Computing Devices in a k8s cluster.** It includes everything you would expect, such as:
+HAMi is a sandbox and [landscape](https://landscape.cncf.io/?item=orchestration-management--scheduling-orchestration--hami) project of  
+[Cloud Native Computing Foundation](https://cncf.io/)(CNCF), 
+[CNAI Landscape project](https://landscape.cncf.io/?group=cnai&item=cnai--general-orchestration--hami).
 
-***Device sharing***: Each task can allocate a portion of a device instead of the entire device, allowing a device to be shared among multiple tasks.
 
-***Device Memory Control***: Devices can be allocated a specific device memory size (e.g., 3000M) or a percentage of the whole GPU's memory (e.g., 50%), ensuring it does not exceed the specified boundaries.
+## Device virtualization
 
-***Device Type Specification***: You can specify the type of device to use or avoid for a particular task by setting annotations, such as "nvidia.com/use-gputype" or "nvidia.com/nouse-gputype".
+HAMi provides device virtualization for several heterogeneous devices including GPU, by supporting device sharing and device resource isolation. For the list of devices supporting device virtualization, see [supported devices](#supported-devices)
 
-***Easy to use***: You don't need to modify your task YAML to use our scheduler. All your jobs will be automatically supported after installation. Additionally, you can specify a resource name other than "nvidia.com/gpu" if you prefer.
+### Device sharing
 
-## Major Features
+- Allows partial device allocation by specifying device core usage.
+- Allows partial device allocation by specifying device memory.
+- Imposes a hard limit on streaming multiprocessors.
+- Requires zero changes to existing programs.
+- Support [dynamic-mig](docs/dynamic-mig-support.md) feature, [example](examples/nvidia/dynamic_mig_example.yaml)
 
-- Hard Limit on Device Memory.
+<img src="./imgs/example.png" width = "500" /> 
 
-A simple demostration for Hard Limit:
-A task with the following resources.
+### Device Resources Isolation
 
-```
+A simple demonstration of device isolation:
+A task with the following resources will see 3000M device memory inside container:
+
+```yaml
       resources:
         limits:
-          nvidia.com/gpu: 1 # requesting 1 vGPU
-          nvidia.com/gpumem: 3000 # Each vGPU contains 3000m device memory
+          nvidia.com/gpu: 1 # declare how many physical GPUs the pod needs
+          nvidia.com/gpumem: 3000 # identifies 3G GPU memory each physical GPU allocates to the pod
```

Comment:
@archlitchi 
1. allocates -> allocated.
2. I think the subject should be the resource name, so declare should be declares